dev/pre-commit

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# File generated by pre-commit: https://pre-commit.com
+# ID: 138fd403232d2ddd5efb44317e38bf03
```

Comment:
Remove these unhelpful hints. We don't have to rely on this is to modify and regenerate this file.